docs/resources/fgs_function.md

```diff
@@ -381,7 +435,8 @@ $ terraform import huaweicloud_fgs_function.test <id>
 
 Note that the imported state may not be identical to your resource definition, due to the attribute missing from the
 API response. The missing attributes are:
-`app`, `func_code`, `agency`, `tags"`.
+`app`, `func_code`, `agency`, `tags"`, `package`, `idle_mode`, `qualifier_name`,  `qualifier_type`,
+`reserved_instance_count`, `tactics_config`.
```

Comment:
Field `qualifier_type` is Computed, it is unnecessary to ignore it.

Reply:
**qualifier_type ** is changed Computed, the qualifier_name  obtained during the import is empty, the value cannot be get in API request.